pypeit/par/pypeitpar.py

```diff
@@ -1660,33 +1681,38 @@ def __init__(self, telgridfile=None, sn_clip=None, resln_guess=None, resln_frac_
 
 
         # Force resln_frac_bounds to be a tuple
-        if pars['resln_frac_bounds'] is not None and not isinstance(pars['resln_frac_bounds'], tuple):
-            try:
-                pars['resln_frac_bouncs'] = tuple(pars['resln_frac_bounds'])
-            except:
-                raise TypeError('Could not convert provided resln_frac_bounds to a tuple.')
-
+        tuple_force(pars,'resln_frac_bounds')
         defaults['resln_frac_bounds'] = (0.5,1.5)
         dtypes['resln_frac_bounds'] = tuple
         descr['resln_frac_bounds'] = 'Bounds for the resolution fit optimization which is part of the telluric model. ' \
                                      'This range is in units of the resln_guess, so the (0.5, 1.5) would bound the ' \
                                      'spectral resolution fit to be within the range ' \
                                      'bounds_resln = (0.5*resln_guess, 1.5*resln_guess)'
 
-        # Force pix_shisft_bounds to be a tuple
-        if pars['pix_shift_bounds'] is not None and not isinstance(pars['pix_shift_bounds'], tuple):
-            try:
-                pars['pix_shift_bounds'] = tuple(pars['pix_shift_bounds'])
-            except:
-                raise TypeError('Could not convert provided pix_shift_bounds to a tuple.')
-
+        tuple_force(pars,'pix_shift_bounds')
         defaults['pix_shift_bounds'] = (-5.0,5.0)
         dtypes['pix_shift_bounds'] = tuple
         descr['pix_shift_bounds'] = ' Bounds for the pixel shift optimization in telluric model fit in units of pixels. ' \
                                     'The atmosphere will be allowed to shift within this range during the fit.'
 
-
-        defaults['maxiter'] = 3
+        tuple_force(pars,'delta_coeff_bounds')
```

Comment:
check with @kbwestfall that this won't have 
unintended consequences

Reply:
It does not, was already being done before I just made it a method.